Containers/borgbackup/Dockerfile

```diff
@@ -13,11 +13,8 @@ RUN set -ex; \
 
 VOLUME /root
 
-COPY start.sh /usr/bin/
-COPY backupscript.sh /
-RUN chmod +x /usr/bin/start.sh; \
-    chmod +x /backupscript.sh
+COPY --chmod=775 start.sh /usr/bin/start.sh
+COPY --chmod=775 backupscript.sh /backupscript.sh
 
-USER root
 ENTRYPOINT ["start.sh"]
```

Comment:
```suggestion
COPY --chmod=770 *.sh /

ENTRYPOINT ["/start.sh"]
```

Reply:
why not 775?

Reply:
can you please apply the suggestion, because: `Applying suggestions on deleted lines is currently not supported.`